ui/components/AutomationsTable.tsx

```diff
@@ -140,7 +153,17 @@ function AutomationsTable({ className, automations, hideSource }: Props) {
 
   return (
     <URLAddressableTable
-      fields={fields}
+      fields={
+        flags.WEAVE_GITOPS_FEATURE_TENANCY === "true"
```

Comment:
I've put this at the end as having something like: 
```
flags.WEAVE_GITOPS_FEATURE_TENANCY === "true" &&  { label: "Tenant", value: "tenant" }
```
in the middle of the list resulted in an empty visible column when the flag is switched off. Any ideas on how to manage better welcomed

Reply:
The magic voodoo might be something like:
```js
[
   { label: "namespace" },
   ...(flags.WEAVE_GITOPS_FEATURE_TENANCY === "true" ? [{ label: "tenant" }] : []),
   { label: "updated at" },
]
```